neuralnetwork/machinelearning/TrainMethod.java

```diff
@@ -0,0 +1,16 @@
+package neuralnetwork.machinelearning;
+
+import neuralnetwork.initializemethod.HeInitialization;
+import neuralnetwork.initializemethod.InitializeMethod;
+import neuralnetwork.initializemethod.XavierInitialization;
+
+public enum TrainMethod {
```

Comment:
We can also use 'He' by just using 'new HeInitialization', right? I can't understand this properly